pex/venv/pex.py

```diff
@@ -417,7 +417,17 @@ def sys_executable_paths():
                 )
 
             os.environ["VIRTUAL_ENV"] = venv_dir
-            sys.path.extend(os.environ.get("PEX_EXTRA_SYS_PATH", "").split(os.pathsep))
+
+            # A Python interpreter always inserts the CWD at the head of the sys.path.
+            sys.path.insert(0, "")
```

Comment:
This is almost certainly a bug. Inserting CWD here before determining below if we're actually running as an interpreter. The equivalent PEX zipapp code only triggers when running as an interpreter.

Reply:
https://docs.python.org/3/library/sys.html#sys.path lists the cases for which paths get prepended (including `""`)

Reply:
https://github.com/pantsbuild/pex/issues/1983